api/apiclient.go

```diff
@@ -428,10 +423,6 @@ func (st *state) connectStream(path string, attrs url.Values, extraHeaders http.
 	dialer := &websocket.Dialer{
 		Proxy:           proxy.DefaultConfig.GetProxy,
 		TLSClientConfig: st.tlsConfig,
-		// In order to deal with the remote side not handling message
-		// fragmentation, we default to largeish frames.
-		ReadBufferSize:  websocketFrameSize,
-		WriteBufferSize: websocketFrameSize,
```

Comment:
The dialer documentations says
```
// ReadBufferSize and WriteBufferSize specify I/O buffer sizes in bytes. If a buffer
// size is zero, then a useful default size is used. The I/O buffer sizes
// do not limit the size of the messages that can be sent or received.
```
The default buffers are 
```
defaultReadBufferSize  = 4096
defaultWriteBufferSize = 4096
```
if we are OK with those sizes go ahead.